pyvista/__init__.py

```diff
@@ -9,23 +9,272 @@
 # Load default theme.  Must be loaded first.
 from pyvista._version import __version__
 from pyvista.plotting import *
-from pyvista.utilities import *
-from pyvista.core import *
+from pyvista.utilities import (
+    # arrays,
+    cells,
+    # common,
+    # errors,
+    # features,
+    fileio,
+    # geometric_objects,
+    helpers,
```

Comment:
This is here due to `doc/api/core/helpers.rst` so it's probably here for good, `pyvista.helpers` is likely to be used downstream.